tap_gitlab/schemas/groups.json

```diff
@@ -0,0 +1,59 @@
+{
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "integer"
+        },
+        "name": {
+            "type": ["null", "string"]
+        },
+        "path": {
+            "type": ["null", "string"]
+        },
+        "description": {
+            "type": ["null", "string"]
+        },
+        "visibility_level": {
+            "type": ["null", "integer"]
+        },
+        "lfs_enabled": {
+            "type": ["null", "boolean"]
+        },
+        "avatar_url": {
+            "type": ["null", "string"]
+        },
+        "web_url": {
+            "type": ["null", "string"]
+        },
+        "request_access_enabled": {
+            "type": ["null", "boolean"]
+        },
+        "full_name": {
+            "type": ["null", "string"]
+        },
+        "full_path": {
+            "type": ["null", "string"]
+        },
+        "id": {
```

Comment:
This appears to be duplicating / shadowing the `id` element higher up on line 4.

Reply:
My bad, I've corrected it.